### ISSUE TITLE
Fails to `npm install`

### ISSUE THREAD

**Issue by skube** [adorableio/ywebca-site#15]
This could be just me but when I clone and run `npm install` I get a lot of errors including:
```
npm ERR! Failed at the ywebca-site@0.0.1 postinstall script 'bower install && gulp'.
npm ERR! This is most likely a problem with the ywebca-site package,
npm ERR! not with npm itself.
```

**Comment by itsthatguy** [comment]
Could you paste the full error?

**Comment by chriscoyier** [comment]
I remember I had to manually create a `bower_components` folder at the root

**Comment by bigtiger** [comment]
Reproduced the error and manually creating a `bower_components` folder solved the issue. Will work with @itsthatguy on a solution.

**Comment by skube** [comment]
Yep. I created the `bower_components` folder manually and also had to run as root/administrator. Worked now. Thx.

**Comment by itsthatguy** [comment]
Could you remove the `bower_components` folder, and try using #16 to see if that works for you?
